bin/crx.js

```diff
@@ -98,14 +113,15 @@ function pack(dir, program) {
 
   var crx = new ChromeExtension({
     rootDirectory: input,
-    maxBuffer: program.maxBuffer
+    maxBuffer: program.maxBuffer,
+    version: program.formatVersion
   });
 
   readFile(keyPath)
     .then(null, function(err) {
       // If the key file doesn't exist, create one
       if (err.code === "ENOENT") {
-        return generateKeyFile(keyPath);
+        return generateKeyFile(keyPath, program).then(() => readFile(keyPath));
```

Comment:
Why the extra `then(() => readFile(keyPath))` here?

Reply:
@arkon because we need to read key data from newly created file. `generateKeyFile` resolves to whatever is returned by `writeFile` (which is info about how many bytes where written, IIRC), not to key data.